modules/apps/object/object-test/src/testFunctional/tests/ObjectSalesforce.testcase

```diff
@@ -21,4 +21,84 @@ definition {
 		}
 	}
 
+	@description = "LPS-185427 - Verify CRUD functionality with an imported custom object using Salesforce as the storage type."
```

Comment:
Verify CRUD functionality with an imported custom object using Salesforce as the storage type

Reply:
We don't need a period at the end